RideShare/src/app/services/AllUsersServices/all-users.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from 'src/app/models/user.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AllUsersService {
+  /**
+   * Server and port number of the request
+   */
+  readonly port = 'http://localhost:3001';
```

Comment:
Move to environment variable

Reply:
Changed to environment variable in environments.ts.